extensions/arc/src/localizedConstants.ts

```diff
@@ -164,13 +164,21 @@ export const nodes = localize('arc.nodes', "nodes");
 export const workerNodes = localize('arc.workerNodes', "Worker Nodes");
 export const coordinatorNode = localize('arc.coordinatorNode', "Coordinator Node");
 export const storagePerNode = localize('arc.storagePerNode', "storage per node");
-export const workerNodeCount = localize('arc.workerNodeCount', "Worker node count:");
+export const workerNodeCount = localize('arc.workerNodeCount', "Worker node count");
```

Comment:
Moved the : to the code - it doesn't need to be localized anyways and would be weird in the screen reader text